packages/discovery-provider/plugins/pedalboard/apps/solana-relay/src/routes/relay/assertRelayAllowedInstructions.ts

```diff
@@ -144,7 +144,10 @@ const assertAllowedAssociatedTokenAccountProgramInstruction = async (
             instr.keys.destination.pubkey
           )
       )
-    if (wallet) {
+    if (
+      wallet &&
+      matchingCreateInstructions.length !== matchingCloseInstructions.length
```

Comment:
can you explain this to me? i thought we were now just allowing create ATA instructions through with a rate limit. Why do we need to check the length of the create/close instructions still?

Reply:
I'm leaving behind the opportunity for matching close/opens without incrementing your rate limit counter. Kinda superfluous if we don't end up using it anywhere though, I admit